common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4673,6 +4673,41 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_SHOW_CREATE_TABLE" : {
+    "message" : [
+      "Unsupported show create table command:"
+    ],
+    "subClass" : {
+      "ON_SPARK_DATA_SOURCE_TABLE_WITH_AS_SERDE" : {
+        "message" : [
+          "<table> is a Spark data source table. Use `SHOW CREATE TABLE` without `AS SERDE` instead."
+        ]
+      },
+      "ON_TEMPORARY_VIEW" : {
+        "message" : [
+          "SHOW CREATE TABLE is not supported on a temporary view: <table>."
+        ]
+      },
+      "ON_TRANSACTIONAL_HIVE_TABLE" : {
+        "message" : [
```

Comment:
I suggest making the message for `ON_TRANSACTIONAL_HIVE_TABLE` and `ON_SPARK_DATA_SOURCE_TABLE_WITH_AS_SERDE` more consistent in the pattern.